File: results/java/LEXICOGRAPHICALLY_MINIMUM_STRING_ROTATION.cpp
string f_filled ( string str ) {
  int n = str . length ( ) ;
  string arr [ n ] ;
  string concat = str + str ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) arr [ i ] = concat . substr ( i , n ) ;
  sort ( arr , arr + n ) ;
  return arr [ 0 ] ;
}

File: TransCoder_tokenized_test_set_functions/java/LEXICOGRAPHICALLY_MINIMUM_STRING_ROTATION.java
static String minLexRotation ( String str ) {
  int n = str . length ( );
  String arr [ ] = new String [ n ];
  String concat = str + str;
  for ( int i = 0;
  i < n;
  i ++ ) {
    arr [ i ] = concat . substring ( i, i + n );
  }
  Arrays . sort ( arr );
  return arr [ 0 ];
}